src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs

```diff
@@ -3,6 +3,8 @@
 
 using System;
 using System.Collections.Generic;
+using System.Drawing;
```

Comment:
nit: unused

Reply:
Ah yes, after moving the function these were left. I removed drawing and regularexpression. This file should have no changes anymore for this PR.